fandogh_cli/fandogh_client/__init__.py

```diff
@@ -326,3 +305,50 @@ def deploy_manifest(manifest):
         raise get_exception(response)
     else:
         return response.json()
+
+
+def _generate_manifest_yaml(image, version, name, port, envs, hosts, internal, registry_secret, image_pull_policy,
+                            internal_ports):
+    yaml = dict()
+
+    if internal:
+        yaml['kind'] = 'InternalService'
+    else:
+        yaml['kind'] = 'ExternalService'
+
+    if name:
+        yaml['name'] = name
+
+    spec = dict()
+
+    if image:
+        spec['image'] = '{}:{}'.format(image, version)
+
+    env_lis = []
+    if version:
+        env_lis.append({'name': 'VERSION', 'value': version})
```

Comment:
why is it needed?

Reply:
VERSION in manifest is hard coded so we have to do it ourselves
but the check statement is redundant.